01_version_control/my_favorite_neat_little_git_trick_demo.md

```diff
@@ -26,3 +26,15 @@
 - One can create empty commits, i.e. a commit without any changes, using the `--allow-empty` options. This can be useful for debugging, for example. The full command could look like this: `git commit --allow-empty -m 'This commit is empty'`
 - You can generate a list of contributors sorted by the number of contributions using `git shortlog --summary --numbered --email.
 - If you have some contributors with several different email addresse and/or configured names, you can merge them via aliases in a file called [`.mailmap`](https://git-scm.com/docs/gitmailmap) that should reside in the root of the repository.
+
+## Git Lense
+
+- [Git Lens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)  extends the Git capabilities built into Visual Studio Code. It helps you visualize code authorship at a glance using Git Blame annotations and Code Lens, seamlessly navigate and explore Git repositories, gain valuable insights with powerful comparison commands, and more.
+
+- The Git Lens extension is one of the most popular in the community and is also the most powerful. In most cases, its functionality can replace either of the previous two extensions.
+
+- For blame information, a subtle message appears to the right of the line you're working on, letting you know the change was made when it was made and the associated commit message. Some additional information is shown when you hover over this message, like the code change itself, the timestamp, and more.
+
+- For Git history information, this extension offers a variety of features. You can easily access various options, including viewing file history, comparing to previous versions, opening a specific revision, and more. To open these options, you can click the text in the bottom status bar that contains the author who edited the line of code and how long it has been since it was edited.
+
+- This extension is packed with functionality and will take you a while to absorb everything it offers.
```

Comment:
Could you please move it up under the "Tricks WT 2022/23" and make it a level 3 header?